getstation/catalog
# [ETF-186] Button UI Component (ex AppItem/Category)

# Add `<AppItem/>` to catalog
## Linear: [ETF-185](https://linear.app/issue/ETF-185/`lessappitemgreater`-ui-component)
## Spec: [Notion](https://www.notion.so/stationhq/Technical-Specs-f216f9b7287a414ea9c1f84e1d9fffa7#85ab0a7b336a4bfbb7ff8ea2b11b8dbe)

### Test instructions
- `yarn && yarn start`
- make sure that Component is visually identical to [design](https://www.notion.so/stationhq/Technical-Specs-f216f9b7287a414ea9c1f84e1d9fffa7#85ab0a7b336a4bfbb7ff8ea2b11b8dbe)
- Make sure that all editable properties work fine.
